homeassistant/components/weatherflow_cloud/sensor.py

```diff
@@ -176,35 +317,154 @@ async def async_setup_entry(
 ) -> None:
     """Set up WeatherFlow sensors based on a config entry."""
 
-    coordinator: WeatherFlowCloudDataUpdateCoordinator = hass.data[DOMAIN][
-        entry.entry_id
-    ]
+    coordinators: WeatherFlowCoordinators = hass.data[DOMAIN][entry.entry_id]
+    rest_coordinator = coordinators.rest
+    wind_coordinator: WeatherFlowCloudDataCallbackCoordinator[
+        EventDataRapidWind, EventDataRapidWind, RapidWindListenStartMessage
+    ] = coordinators.wind
+    observation_coordinator: WeatherFlowCloudDataCallbackCoordinator[
+        WebsocketObservation, WebsocketObservation, ListenStartMessage
+    ] = coordinators.observation
 
-    async_add_entities(
-        WeatherFlowCloudSensor(coordinator, sensor_description, station_id)
-        for station_id in coordinator.data
+    entities: list[SensorEntity] = []
+    entities.extend(
```

Comment:
```suggestion
    entities: list[SensorEntity] = [
```
first one can be combined